python/ray/experimental/sgd/utils.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from contextlib import closing
+import numpy as np
+import socket
+import time
+
+
+class TimerStat(object):
```

Comment:
should this be removed from the Torch utils?

Reply:
I think it is better to reflect in separate branch.

Reply:
(moved)